### PR TITLE
api: extracts resampling method as config variable

### LINKED ISSUE #17: Pillow resampling method should be a config variable
The choice of resampling method depends on the application of this module, and should be configurable through a config setting.

### PR DESCRIPTION
**Pull request by blixhavn**
- BETTER Adds the config variable `IIIF_RESAMPLING` with default
  `PIL.Image:NEAREST` (closes #17)

Signed-off-by: Øystein Blixhavn oystein@blixhavn.no

### CODE REVIEW

**Comment by jirikuncar** [comment]
```
api: configurable default resample value

* BETTER Makes default resample value of resize resize method configurable using
  `IIIF_RESIZE_RESAMPLE` config variable with default `PIL.Image:NEAREST`. 
  (closes #17)

Signed-off-by: Øystein Blixhavn <oystein@blixhavn.no>
```

File: flask_iiif/config.py [hunk 1]
```diff
@@ -58,6 +58,8 @@
         <http://iiif.io/api/image/2.0/#information-request>`_
 
 """
+# Resampling method
+IIIF_RESAMPLING = 'PIL.Image:NEAREST'
```
Comment (jirikuncar) [inline flask_iiif/config.py]: what about calling it `IIIF_RESIZE_RESAMPLE`?

File: flask_iiif/api.py [hunk 2]
```diff
@@ -134,6 +136,8 @@ def resize(self, dimensions, resample=Image.NEAREST):
 
         """
         real_width, real_height = self.image.size
+        if resample is None:
+            resample = import_string(current_app.config['IIIF_RESAMPLING'])
```
Comment (jirikuncar) [inline flask_iiif/api.py]: Can you check if it is string type before importing it? Then you can define the value directly in `config.py` as `Image.NEAREST` (`from PIL import Image`).

**Comment by blixhavn** [comment]
@drjova This now builds

**Pull request merged**